### DISCUSSION TITLE
請求支援皮膚的自動切換深色模式功能

### DISCUSSION THREAD

**Discussion by developor** [rime/home#1448] [General]
trime 已經可以了，pc 上小狼毫還不知道怎麼弄

**Comment by fxliang** [comment]
https://github.com/rime/weasel/releases/tag/latest
试下最新构建，已有了，方案里也可定义呢
`style/color_scheme_dark: lost_temple`

**Reply by developor** [reply]
多謝大佬指教，成功了 ！

**Reply by treemoons** [reply]
請問，托盤圖標是否也能跟隨深色主題一起變換？「中」那個圖標，想要變成把色模式下用黑色字體他，深色模式用白色字體，而不用其他底色。 
例如在方案中設置： 
`icon_dark: ico_dark.ico`
`ascii_icon_dark: ico_ascii_dark.ico`
